proto/tripcommon.proto

```diff
@@ -58,11 +58,11 @@ message Location {
   optional string url = 11;
   optional string date_time = 12;
   optional SideOfStreet side_of_street = 13;
-  optional uint32 heading_tolerance = 14 [default = 60];
-  optional uint32 node_snap_tolerance = 15 [default = 5];
+  optional uint32 heading_tolerance = 14;
+  optional uint32 node_snap_tolerance = 15;
   optional uint64 way_id = 16;
-  optional uint32 minimum_reachability = 17 [default = 0];
-  optional uint32 radius = 18 [default = 0];
```

Comment:
we no longer let the pbf default these because that nukes the ability to default them via config